Tutorial Screen (#63)
SamGonChec/449_The_Gifted_Chimpanzees: NineMensMorris/src/tutorial.txt

Players take turns to play a piece of their own colour on any unoccupied point until all eighteen pieces have been played. 
After that, play continues alternately but each turn consists of a player moving one piece along a line to an adjacent empty point.

> During both of these phases, whenever a player have three of their pieces form a line they achieved a mill. 

Reviewer: ```suggestion
During each turn, whenever a player has three of their pieces in a line, they form a mill. 
```

That player then immediately removes from the board one piece belonging to the opponent that does not form part of a mill. 

> If all the opponents pieces form mills then an exception is made and the player is allowed to remove any piece. 

Reviewer: ```suggestion
If all of the opponent's pieces form mills, then the player is allowed to remove any one of the opponent's pieces. 
```
